src/cpptoolsconfigprovider.ts

```diff
@@ -0,0 +1,120 @@
+import path = require("path");
```

Comment:
We should document this in the README.md somewhere.

Reply:
Done